src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/service/IncentiveStoreService.kt

```diff
@@ -0,0 +1,89 @@
+package uk.gov.justice.digital.hmpps.incentivesapi.service
```

Comment:
This is a new service to wrap the insert/updates to incentive data with a Transactional boundary